### ISSUE TITLE
kubectl describe hpa broken

### ISSUE THREAD

**Issue by piosz** [kubernetes/kubernetes#21669]
```
$ kubectl describe hpa php-apache
...
Min pods:			833358050696
...
```

**Comment by AdoHe** [comment]
@piosz could you provide version info? I would take a look. Thanks very much.

**Comment by piosz** [comment]
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.7", GitCommit:"e4e6878293a339e4087dae684647c9e53f1cf9f0", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2+", GitVersion:"v1.2.0-alpha.7.939+d00c650b28843b", GitCommit:"d00c650b28843b1edf910c28bd3ec92f5b32f662", GitTreeState:"clean"}
```
Thanks!

**Comment by piosz** [comment]
Ups, I've already fixed this #16025. Should be cherry-picked to release-1.1

**Comment by piosz** [comment]
This only happens with kubectl in version 1.1 so removing milestone.

**Comment by piosz** [comment]
fixed by #21670, will be released in 1.1.8
